### ISSUE TITLE
Support columns with `server_default`

### ISSUE THREAD

**Issue by kreathon** [imankulov/alembic-enums#2]
At the moment, the following error is produced:

```
sqlalchemy.exc.ProgrammingError: (psycopg2.errors.DatatypeMismatch) default for column "abc" cannot be cast automatically to type _tmp_cdef_1720

```

My first idea was to extend `alembic_enums.Column` to support something like `new_server_default` (which could be NOT_SET, None or a value).

Have you thought about supporting something like that?

**Comment by imankulov** [comment]
Hey @kreathon, thanks for the report.

Can you please take a look at the PR #3 and let me know if it works for you. As outlined in the documentation in the updated README, you can use the following syntax.

```python
column = Column(
    "resources",
    "state",
    server_default=Change(old="enabled", new="active"),
)
```

**Comment by kreathon** [comment]
Thank you for addressing this so quickly, I will leave a comment on the PR :slightly_smiling_face: .

**Comment by imankulov** [comment]
Thanks a lot for spotting the issue and reviewing the fix, @kreathon 

The final syntax for defining columns:

```python
column = Column(
    "resources",
    "state",
    old_server_default="enabled",
    new_server_default="active",
)
```

The fix is available in the version [0.2.0](https://pypi.org/project/alembic-enums/0.2.0/).
